app/helpers/enterprises_helper.rb

```diff
@@ -88,4 +88,11 @@ def shop_trial_expired?(enterprise)
   def remaining_trial_days(enterprise)
     distance_of_time_in_words(Time.zone.now, enterprise.shop_trial_start_date + Spree::Config[:shop_trial_length_days].days)
   end
+
+  # Enable embeddeding within iframes on external sites.
+  # Call with :after_filter on controller actions.
+  def enable_embedded_shopfront
```

Comment:
We could at least create dummy html with iframe to check if it doesn't render layout.
Other option would be to create dummy app (i.e. on heroku) which would accept iframe source as GET param. I know, going outside testing environment into internet might be a bit costly.